velox/functions/sparksql/Register.cpp

```diff
@@ -242,6 +242,8 @@ void registerFunctions(const std::string& prefix) {
   registerFunction<Sha2HexStringFunction, Varchar, Varbinary, int32_t>(
       {prefix + "sha2"});
   registerFunction<CRC32Function, int64_t, Varbinary>({prefix + "crc32"});
+  registerFunction<Empty2NullFunction, Varchar, Varchar>(
+      {prefix + "empty2null"});
```

Comment:
Perhaps add documentation for this function in https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/string.rst.

Reply:
added, thanks!